devtools/gradle/gradle-model/src/main/java/io/quarkus/gradle/tooling/dependency/DependencyUtils.java

```diff
@@ -377,7 +378,8 @@ private static Dependency createDeploymentProjectDependency(DependencyHandler ha
                     ped.getDeploymentModule().getName(),
                     ped.getDeploymentModule().getVersion().toString());
         } else if (ped.getDeploymentModule() instanceof ProjectInternal) {
-            return handler.create(new DefaultProjectDependency((ProjectInternal) ped.getDeploymentModule(), true));
+            return handler.create(new DefaultProjectDependency((ProjectInternal) ped.getDeploymentModule(), true,
+                    DefaultTaskDependencyFactory.withNoAssociatedProject()));
```

Comment:
@snazy just to be sure, the new constructor you used is present in older Gradle versions?

Reply:
Yep, also there in older Gradle versions.

Reply:
Gradle 8.5.0 is the minimum version (older ones don't have that DefaultProjectDependency class)

Reply:
So no change in "minimum Gradle version" requirement